src/Joblify.Core.Api/AutoMapperProfile.cs

```diff
@@ -2,10 +2,18 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using AutoMapper;
+using Joblify.Core.Data.Models;
+using Joblify.Core.Login.Dto;
 
 namespace Joblify.Core.Api
 {
-    public class AutoMapperProfile
+    public class AutoMapperProfile: Profile
     {
+        public AutoMapperProfile()
+        {
+            CreateMap<User, LoginDto>();
+            CreateMap<LoginDto, User>();
+        }
     }
 }
```

Comment:
Consider putting this file in something like "Infrastructure" folder to avoid mess in Api project structure